src/Http/HttpClientFactory.php

```diff
@@ -87,7 +93,7 @@ public static function nationalCloud(string $nationalCloud = NationalCloud::GLOB
      */
     public static function clientConfig(array $config): HttpClientFactory {
```

Comment:
```suggestion
    public static function setClientConfig(array $config): HttpClientFactory {
```

Reply:
Updated naming of all setters methods in this class